string msgType
int64 timestamp
string version

float64 mileage
float64 batterySOH
float64 batterySOC
float64 batteryVoltage
float64 batteryCurrent
int32 charge_state

float32 vehSpeed    
float32 pointHA
float64 pointX
float64 pointY
string workPointid

int32 steerControlMode
int32 vehCtrlMode	
int32 autoDriveEnable	

float32 pitchAngle
float32 azimuthAngle

string taskId
int32 taskStatus
string errCode

